dev/git

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# github.com/gravitational/teleport contains private git submodules that can't be
+# checkouted by dep, so we have to use git wrapper that ignores submodules
+
+if [ -z $_GIT_ORIGIN ]; then
+  echo "Don't use this wrapper directly";
+  exit 1;
+fi
+
+if [ $1 == "submodule" ]; then
+  echo "Ignoring submodules";
```

Comment:
It's a neat hack 👍 
Though i'm not sure that it's okay to include these scripts in the repo.
Another thing that bothers me: need to limit disabling of `git submodule` command *only* to `github.com/gravitational/teleport`. Other repos could use `git submodule` too.

Reply:
`go mod` doesn't support git submodules, so I don't think any of pkg would use them. I'd offer to keep it as is until we run into a problem.